src/lib.rs

```diff
@@ -91,3 +96,12 @@ pub unsafe fn JS_ARGV(_cx: *mut JSContext, vp: *mut JSVal) -> *mut JSVal {
 pub unsafe fn JS_CALLEE(_cx: *mut JSContext, vp: *mut JSVal) -> JSVal {
     *vp
 }
+
+// This is measured properly by the heap measurement implemented in SpiderMonkey.
+impl<T: Copy + GCMethods<T>> HeapSizeOf for Heap<T> {
+    fn heap_size_of_children(&self) -> usize {
+        0
+    }
+}
+known_heap_size!(0, JSVal);
```

Comment:
Okay. If I'm going to do that, then where is the `bindgen` program that `gen.py` calls? It seems to be generating the `jsapi::Value` struct.